app/services/google/search_service.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+module Google
+  class SearchService
+    HOST = 'www.google.com'
+
+    # rubocop:disable Layout/LineLength
+    USER_AGENT = [
+      # MacOS Chrome
+      'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/114.0.0.0 Safari/537.36',
+      # Window 10 Chrome
+      'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/109.0.0.0 Safari/537.36',
+      # MacOS Firefox
+      'Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:77.0) Gecko/20100101 Firefox/77.0',
+      # Linux Firefox
+      'Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:15.0) Gecko/20100101 Firefox/15.0.1',
+      # iOS Safari
+      'Mozilla/5.0 (iPhone9,4; U; CPU iPhone OS 10_0_1 like Mac OS X) AppleWebKit/602.1.50 (KHTML, like Gecko) Version/10.0 Mobile/14A403 Safari/602.1',
+      # Android Chrome
+      'Mozilla/5.0 (Linux; Android 12; SM-G973U) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/112.0.0.0 Mobile Safari/537.36'
+    ].freeze
+    # rubocop:enable Layout/LineLength
+
+    private_constant :HOST, :USER_AGENT
+
+    def initialize(keyword)
+      @keyword = keyword
+    end
+
+    # rubocop:disable Metrics/MethodLength
+    def search!
+      uri = URI::HTTPS.build host: HOST, path: '/search', query: { q: keyword }.to_query
+      conn = http_client uri
```

Comment:
It seems like you missed my previous comment about not using the abbreviated name coming from the Phoenix world 😅 

```suggestion
      client = http_client(uri)
```

Reply:
Fixed in 9b3698824c9292c74247204b22f7df5c0fbed19c